runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/windmill/client/WindmillStream.java

```diff
@@ -41,6 +41,8 @@ public interface WindmillStream {
   /** Returns when the stream was opened. */
   Instant startTime();
 
+  boolean isClosed();
```

Comment:
add comment, does this just reflect if close() is called? or if server closed stream also?

Reply:
added comment